README.md

```diff
@@ -103,6 +104,60 @@ The following snippet shows an example `build.gradle` with APK splits enabled.
 
 The adoption of APK splits requires developers to submit multiple APKs to the Play Store. Refer to [Google’s documentation](https://developer.android.com/google/play/publishing/multiple-apks.html) for how to support this in your application.
 
+## Troubleshooting Audio
+The following sections provide guidance on how to ensure optimal audio quality in your applications.
+
+### Configuring [AudioManager](https://developer.android.com/reference/android/media/AudioManager.html)
+The following snippet shows how to configure `AudioManager` for optimal experience when sharing
+audio to a `Room`.
+
+    private void configureAudio(boolean enable) {
+        if (enable) {
+            previousAudioMode = audioManager.getMode();
+            // Request audio focus before making any device switch.
+            audioManager.requestAudioFocus(null, AudioManager.STREAM_VOICE_CALL,
+                    AudioManager.AUDIOFOCUS_GAIN_TRANSIENT);
+            /*
+             * Use MODE_IN_COMMUNICATION as the default audio mode. It is required
+             * to be in this mode when playout and/or recording starts for the best
+             * possible VoIP performance. Some devices have difficulties with
+             * speaker mode if this is not set.
+             */
+            audioManager.setMode(AudioManager.MODE_IN_COMMUNICATION);
+            /*
+             * Always disable microphone mute during a WebRTC call.
+             */
+            previousMicrophoneMute = audioManager.isMicrophoneMute();
+            audioManager.setMicrophoneMute(false);
+        } else {
+            audioManager.setMode(previousAudioMode);
+            audioManager.abandonAudioFocus(null);
+            audioManager.setMicrophoneMute(previousMicrophoneMute);
+        }
+    }
+
+### Configuring Hardware Audio Effects
```

Comment:
Why not recommend using the `AudioOptions.Builder()` options when creating the `LocalAudioTrack` instead?